pkg/resources/secondary_database.go

```diff
@@ -0,0 +1,433 @@
+				},
+			}
+		}
+		return diag.FromErr(err)
+	}
+
+	secondaryDatabaseParameters, err := client.Parameters.ShowParameters(ctx, &sdk.ShowParametersOptions{
+		In: &sdk.ParametersIn{
+			Database: secondaryDatabaseId,
+		},
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	replicationDatabases, err := client.ReplicationFunctions.ShowReplicationDatabases(ctx, &sdk.ShowReplicationDatabasesOptions{
+		Like: &sdk.Like{
+			Pattern: sdk.String(secondaryDatabaseId.Name()),
+		},
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	var replicationPrimaryDatabase *sdk.ReplicationDatabase
+	for _, replicationDatabase := range replicationDatabases {
+		replicationDatabase := replicationDatabase
+		if !replicationDatabase.IsPrimary &&
+			replicationDatabase.AccountLocator == client.GetAccountLocator() &&
+			replicationDatabase.Name == secondaryDatabaseId.Name() {
+			replicationPrimaryDatabase = &replicationDatabase
+		}
+	}
+	if replicationPrimaryDatabase == nil {
+		return diag.FromErr(fmt.Errorf("could not find replication database for %s", secondaryDatabaseId.Name()))
+	}
+
+	if err := d.Set("name", secondaryDatabase.Name); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("as_replica_of", sdk.NewExternalObjectIdentifierFromFullyQualifiedName(replicationPrimaryDatabase.PrimaryDatabase).FullyQualifiedName()); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("is_transient", secondaryDatabase.Transient); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("data_retention_time_in_days", []any{map[string]any{"value": secondaryDatabase.RetentionTime}}); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("comment", secondaryDatabase.Comment); err != nil {
+		return diag.FromErr(err)
+	}
+
+	for _, secondaryDatabaseParameter := range secondaryDatabaseParameters {
+		switch secondaryDatabaseParameter.Key {
+		case "MAX_DATA_EXTENSION_TIME_IN_DAYS":
+			maxDataExtensionTimeInDays, err := strconv.Atoi(secondaryDatabaseParameter.Value)
+			if err != nil {
+				return diag.FromErr(err)
+			}
+			if err := d.Set("max_data_extension_time_in_days", []any{map[string]any{"value": maxDataExtensionTimeInDays}}); err != nil {
+				return diag.FromErr(err)
+			}
+		case "EXTERNAL_VOLUME":
+			if err := d.Set("external_volume", secondaryDatabaseParameter.Value); err != nil {
+				return diag.FromErr(err)
+			}
+		case "CATALOG":
+			if err := d.Set("catalog", secondaryDatabaseParameter.Value); err != nil {
+				return diag.FromErr(err)
+			}
+		case "DEFAULT_DDL_COLLATION":
+			if err := d.Set("default_ddl_collation", secondaryDatabaseParameter.Value); err != nil {
+				return diag.FromErr(err)
+			}
+		case "LOG_LEVEL":
+			if err := d.Set("log_level", secondaryDatabaseParameter.Value); err != nil {
+				return diag.FromErr(err)
+			}
+		case "TRACE_LEVEL":
+			if err := d.Set("trace_level", secondaryDatabaseParameter.Value); err != nil {
+				return diag.FromErr(err)
+			}
+		}
```

Comment:
Hmm, not sure about that. There are at least a few parameters not handled here (coming from god knows where). But now I realized, maybe they should be computed values and log on default, because right now the default would be spamming. Also, maybe such values should be a computed map of parameters... 🤔

Reply:
As discussed verbally yesterday, let's add such a map to the datasources now, and we will discuss adding them also to the resources.